test/functions/iterators/engin/paramArgIterator.good

```diff
@@ -0,0 +1,8 @@
+par: 0
```

Comment:
The output of this test always comes in the same order. ser:0,1,2,3, par:0,10,20,30.

Can you fix the order of this .good file and remove the .prediff?